vexination/walker/src/lib.rs

```diff
@@ -61,6 +62,13 @@ pub struct Run {
     /// Additional root certificates for the destination
     #[arg(long = "sender-root-certificates")]
     pub additional_root_certificates: Vec<PathBuf>,
+    /// Path of the HTML output file
+    #[arg(long, default_value = "report.html")]
+    pub output: PathBuf,
```

Comment:
Maybe it makes more sense to just specify a path, and then use that base path for writing out a `report.html` as well as a `report.json`.